cmd/release-controller/sync_release.go

```diff
@@ -272,7 +272,7 @@ func findContainerStatus(statuses []corev1.ContainerStatus, name string) *corev1
 	return nil
 }
 
-func newReleaseJobBase(name, cliImage, pullSecretName string) (*batchv1.Job, string) {
+func newReleaseJobBase(name, namespace, cliImage, pullSecretName string) (*batchv1.Job, string) {
```

Comment:
Is there enough information available when this method is called to have the annotations you're adding?

Reply:
> Is there enough information available when this method is called to have the annotations you're adding?

I think I've commented in the wrong spot.